scripts/trento-support.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+
+readonly ARGS=("$@")
+declare -A VALID_FACILITIES=(
+    ["configuration"]="configuration"
+    ["base"]="base"
+    ["kubernetes"]="kubernetes"
+    ["all"]="all"
+)
+
+indent() { sed 's/^/  /'; }
+
+collect_trento_configuration() {
+    echo "#==[ Configuration File ]===========================#"
+    echo "# /etc/trento/installer.conf"
+    echo "$(</etc/trento/installer.conf)"    
+} &> "$OUTPUT"
+
+collect_base_system() {
+    echo "#==[ Command ]======================================#"
+    echo "# k3s --version"
+    k3s --version
```

Comment:
> It looks good to me @rtorrero
> 
> My unique doubt is whether this new package should be hosted in the helm-charts project. It doesn't look the most appropriate place to be honest. But well, for easiness sake, let's start here and we could move it afterwards if needed

Yes, I had a few doubts about this too, but I thought about the same approach; for now we can keep it on the same repo as it's related to the helm charts (it basically needs to run on the same machine where the helm charts where used to deploy everything) but for the future it could probably have it's own repo.